### PR TITLE
Rename MVQN to IQNIMVJ

### PR DESCRIPTION
**Pull request by Fujikawas**
## Main changes of this PR
Rename the acceleration method MVQN in the code to IQNIMVJ to align with the name in the configuration.

## Motivation and additional information

Part of https://github.com/precice/precice/issues/1463

## Author's checklist

* [x] I used the [`pre-commit` hook](https://precice.org/dev-docs-dev-tooling.html#setting-up-pre-commit) to prevent dirty commits and used `pre-commit run --all` to format old commits.
* [ ] ~~I added a changelog file with make changelog if there are user-observable changes since the last release.~~
* [x] I squashed / am about to squash all commits that should be seen as one.

## Reviewers' checklist

* [ ] ~~Does the changelog entry make sense? Is it formatted correctly?~~
* [x] Do you understand the code changes?

### CODE REVIEW

File: src/acceleration/IQNIMVJAcceleration.hpp [hunk 1]
```diff
@@ -135,12 +135,12 @@ class MVQNAcceleration : public BaseQNAcceleration {
...
 
   // DEBUG
-  //std::fstream _info2;
+  // std::fstream _info2;
   double _avgRank;
 
-  /** @brief: comptes the MVQN update using QR decomposition of V,
-    *        furthermore it updates the inverse of the system jacobian
-    */
+  /** @brief: comptes the IQNIMVJ update using QR decomposition of V,
```
Comment (uekerman) [inline src/acceleration/IQNIMVJAcceleration.hpp:141]:
```suggestion
  /** @brief: computes the IQNIMVJ update using QR decomposition of V,
```

**Review by uekerman — APPROVED** [review]
Looks good to me :+1:

**Comment by uekerman** [comment]
FYI: I changed "partly closes" to "part of" in the PR description to avoid accidental automatic closing of the issue (which would have happened once we had merged here).

File: src/acceleration/test/AccelerationIntraCommTest.cpp [hunk 2]
```diff
@@ -67,16 +67,16 @@ BOOST_AUTO_TEST_CASE(testVIQNILSpp)
   mesh::PtrData displacements(new mesh::Data("dvalues", -1, 1));
   mesh::PtrData forces(new mesh::Data("fvalues", -1, 1));
 
-  if (context.isPrimary()) { //Primary
+  if (context.isPrimary()) { // Primary
```
Comment (BenjaminRodenberg) [inline src/acceleration/test/AccelerationIntraCommTest.cpp:70]: These little changes are really a nightmare when a merge-conflict occurs. Looks to me like our linter has a blind spot here. Can we somehow fix this @fsimonis ?

**Pull request merged**